docs/pages/cloud/introduction.mdx

```diff
@@ -1,16 +1,61 @@
 ---
-title: Cloud
-description: Teleport as a service
-h1: Teleport cloud
+title: Teleport Cloud
+description: Teleport Cloud is a deployment of the Auth and Proxy Services managed by a dedicated team.
```

Comment:
This positions Teleport as a hosted version of Teleport Enterprise. This is technically correct. But stragegically we should design this page for people who haven't heard about Teleport, Teleport Auth and Proxy are not familiar to them. 
How about this:

Teleport cloud is a managed service to provide access to secure infrastructure all over the world without passwords or shared secrets.

Reply:
Since this is PR is a backport and the main branch was already merged to `master`, I have made this change in a new PR (https://github.com/gravitational/teleport/pull/13151).